### PR TITLE
Improve comments

### PR DESCRIPTION
**Pull request by FauziAkram**
Improve comments.
Non-Functional

### CODE REVIEW

**Comment by cj5716** [comment]
I think you need to write "No functional change" to pass CI currently. Even if it doesn't, I still think it would be good to follow normal procedure.

**Comment by Disservin** [comment]
> I think you need to write "No functional change" to pass CI currently. Even if it doesn't, I still think it would be good to follow normal procedure.

you shouldnt need to right now

**Comment by dav1312** [comment]
If you are updating comments, please take this opportunity to update:

https://github.com/official-stockfish/Stockfish/blob/70ba9de85cddc5460b1ec53e0a99bee271e26ece/src/search.cpp#L103-L106

**Comment by FauziAkram** [comment]
What do you guys suggest to replace the part that dav1312 mentioned with?

File: src/search.cpp [hunk 1]
```diff
@@ -100,10 +100,7 @@ namespace {
     return VALUE_DRAW - 1 + Value(thisThread->nodes & 0x2);
   }
 
-  // Skill structure is used to implement strength limit. If we have an uci_elo then
-  // we convert it to a suitable fractional skill level using anchoring to CCRL Elo
-  // (goldfish 1.13 = 2000) and a fit through Ordo derived Elo for a match (TC 60+0.6)
-  // results spanning a wide range of k values.
+  // Skill structure is used to implement strength limit
```
Comment (Disservin) [inline src/search.cpp:103]: This is not what @dav1312 wanted. The skill is no longer anchored to goldfish but instead stash was used. See https://github.com/vondele/Stockfish/commit/a08b8d4e9711c20acedbfe17d618c3c384b339ec

**Review by Disservin — CHANGES_REQUESTED** [review]

File: src/search.cpp (hunk 1 above)
Comment (FauziAkram) [reply src/search.cpp:103]: @Disservin I see, what do you think about the following update?

// Skill structure is used to implement strength limit. If we have a UCI_Elo, we now convert it to a suitable fractional skill level using anchoring to the Stash engine.
// This update is based on a fit of the Elo results for games played between the master at various skill levels and various versions of the Stash engine, all ranked at CCRL.
// Skill 0 .. 19 now covers CCRL Blitz Elo from 1320 to 3190, approximately.

Comment (Disservin) [reply src/search.cpp:103]: >  suitable fractional skill level using anchoring to the Stash engine.

this sounds a bit weird?

Comment (FauziAkram) [reply src/search.cpp:103]: // Skill structure is used to implement strength limit. If we have a UCI_Elo, we now convert it to an appropriate skill level using anchoring to the Stash engine.
// This method is based on a fit of the Elo results for games played between the master at various skill levels and various versions of the Stash engine, all ranked at CCRL.
// Skill 0 .. 19 now covers CCRL Blitz Elo from 1320 to 3190, approximately.

**Review by Disservin — APPROVED** [review]

**Pull request closed without merging**